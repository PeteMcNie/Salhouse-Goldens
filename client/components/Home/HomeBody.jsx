import React from 'react'

import Container from 'react-bootstrap/Container'
import Row from 'react-bootstrap/Row'
import Col from 'react-bootstrap/Col'

function HomeBody () {
  return (
    <>
      <Container>
        <Row className='row'>
          <Col className='leftCol'>
            <h2 className='bodyTitle'>Welcome to Salhouse Golden Retrievers</h2>
            <p>
              I am a dog lover who fell in love with this beautiful breed when I was a child in England.
              If you would like to read the story of the animals in my life please visit the <a href='/mylifewithanimals'>My Life with Animals</a> page.
            </p>
            <p>
              I have had the privilege of sharing the lives of seven beautiful Golden Retrievers, most of whom are shown in the photos at the top of this page.
              Phoenix was my first Golden, and was the reason I wanted to have more of them. I can&apos;t thank her enough for the lessons she taught me and the
              loyalty only a dog can give even when the human is less than perfect.
            </p>
            <p>
              Kula was born in 2003 and was a really loving gentle dog. She was a great retriever, loving nothing more than to swim in the river or
              sea fetching as many sticks as we can throw for her. In 2007, Kula had a beautiful litter of puppies and we kept one female who we
              named Shadow.
            </p>
            <p>
              Shadow is a real character and is such fun to be with. She is so enthusiastic about her people and has the loveliest nature. Shadow
              had a litter of 7 puppies in April 2010 and we have kept one of those, Sherry. We added an item to the News page each day from the
              day the puppies were born to the time they left us. There are loads of photos and videos of these puppies and some stories of them
              growing up. Go to the News page to see these stories and more.
            </p>
            <p>
              Sherry is a very soft Golden, only wanting to please and be with us. She is a beautiful natured member of our family and dabbled in
              obedience for a while and did quite well. Sherry had a litter of two female puppies on 16 June 2015 - Autumn and Tuppence. Only
              having two in the litter made it impossible for us to part with one, so, although we had only wanted to keep one, we found ourselves
              with the two naughties!
            </p>
            <p>
              Tuppence is the smaller of the two sisters, but she is an exceedingly easy dog to train. She’s been taught lots of tricks and is a
              thinker. If you give her the right messages, she will eventually work out what you want her to do. She goes through her whole repertoire
              of tricks until she finds the one you want. I’ve never had a thinking dog like this before and she’s been really fun to train.
            </p>
            <p>
              Autumn is a relatively strong and powerful Golden girl, whose enthusiasm is very contagious and can be hard to contain when she’s out near water
              or when we’ve got a ball thrower. She is totally focused. Autumn had a litter of seven puppies in March 2019, of which we kept one who we called Siren.
            </p>
            <p>
              Siren is a darling. She’s a very showy girl and looks beautiful in the show ring and when out and about with her head up and tail flowing.
              She’s got a zany character and is another fun dog to be around.
            </p>
            <p>
              If you want to find out more about each of these beautiful dogs, please go to their pages to learn more.
              Shadow, Sherry and Autumn are breed champions. Sherry did compete in championship obedience competitions up to Test A level and has gained
              the CDX title. However, when Autumn and Tuppence came along it was too much to continue training Sherry and the puppies and so she is retired from the
              obedience ring. Tuppence is a very smart dog and at the present time is in training and hopefully one day I will get her into the obedience ring. Go to
              their individual pages to learn more about them.
            </p>
            <p>
              Goldens are wonderful pets - they are loyal, friendly and good with children . I hope you like mine and get your own one day!
            </p>
            <p>
              This website has lots of lovely photos of goldens and some other animals that I have owned, I hope you enjoy looking round it.
            </p>
          </Col>
        </Row>
        <hr className='sectionSeparator' />
      </Container>
    </>
  )
}

export default HomeBody
