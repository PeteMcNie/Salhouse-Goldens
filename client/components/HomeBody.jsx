import React from 'react'

function HomeBody () {
  return (
    <>
      <div>
        <h1>Welcome to Salhouse Golden Retrievers</h1>
      </div>
      <div>
        <p>
            I am a dog lover who fell in love with this beautiful breed when I was a child in England.
            If you would like to read the story of the animals in my life please go to the My Life with Animals page.
        </p>
        <p>
            The golden retrievers in the above photograph are Tuppence, Shadow, Kula, Sherry and Autumn, with Phoenix laying
            down in front. Unfortunately our dear sweet Phoenix passed over the Rainbow Bridge in September 2016 at the ripe
            old age of 14 years, 8 months. However, she is still very much in our hearts and minds and it is very hard to think
            about the fact that she is no longer with us. My website will still tell the story of Phoenix because she was the
            most beautiful first golden for me to own and was the reason I wanted to have more of them. I can't thank her enough
            for the lessons she taught me and the loyalty only a dog can give even when the human is less than perfect.
        </p>
      </div>

    </>
  )
}

export default HomeBody
