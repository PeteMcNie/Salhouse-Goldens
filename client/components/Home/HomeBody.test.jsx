import React from 'react'
import { render } from '@testing-library/react'

import HomeBody from './Home'

test('HomeBody component to match snapshot', () => {
  const { container } = render(<HomeBody />)
  expect(container).toMatchSnapshot()
})
