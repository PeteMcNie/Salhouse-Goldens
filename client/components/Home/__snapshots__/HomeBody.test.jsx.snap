// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeBody component to match snapshot 1`] = `
<div>
  <nav
    class="navbar navbar-expand-lg navbar-light bg-light fixed-top"
  >
    <a
      class="navbar-brand"
      href="/"
    >
      <img
        alt="Goldens retrievers, phoenix and kula"
        class="shlogo d-inline-block align-top"
        src="images/logo1.png"
      />
    </a>
    <button
      aria-controls="responsive-navbar-nav"
      aria-label="Toggle navigation"
      class="navbar-toggler collapsed"
      type="button"
    >
      <span
        class="navbar-toggler-icon"
      />
    </button>
    <div
      class="navbar-collapse collapse"
      id="responsive-navbar-nav"
    >
      <div
        class="ml-auto navbar-nav"
      >
        <a
          class="nav-link"
          data-rb-event-key="/"
          href="/"
        >
          Home
        </a>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            Our Goldens
          </a>
        </div>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            Grooming Your Golden
          </a>
        </div>
        <a
          class="nav-link"
          data-rb-event-key="/mylifewithanimals"
          href="/mylifewithanimals"
        >
          My Life With Animals
        </a>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            About Us
          </a>
        </div>
        <a
          class="nav-link"
          data-rb-event-key="/links"
          href="/links"
        >
          Links
        </a>
        <a
          class="nav-link"
          data-rb-event-key="/contact"
          href="/contact"
        >
          Contact
        </a>
      </div>
    </div>
  </nav>
  <div
    class="belowNavbar carousel slide"
  >
    <ol
      class="carousel-indicators"
    >
      <li
        class="active"
      />
      <li />
      <li />
      <li />
      <li />
      <li />
      <li />
      <li />
      <li />
      <li />
      <li />
      <li />
    </ol>
    <div
      class="carousel-inner"
    >
      <div
        class="active carousel-item"
      >
        <img
          alt="First slide"
          class="d-block w-100"
          src="images/homepage/hpN1.jpeg"
        />
      </div>
      <div
        class="carousel-item"
      >
        <img
          alt="second slide"
          class="d-block w-100"
          src="images/homepage/hpN4.jpeg"
        />
      </div>
      <div
        class="carousel-item"
      >
        <img
          alt="third slide"
          class="d-block w-100"
          src="images/homepage/hpN5.jpeg"
        />
        <div
          class="carousel-caption"
        >
          <h3>
            First slide label, do you want this?
          </h3>
          <p>
            Do you want to add further comments below the label?
          </p>
        </div>
      </div>
      <div
        class="carousel-item"
      >
        <img
          alt="Forth slide"
          class="d-block w-100"
          src="images/homepage/hpN9.jpeg"
        />
      </div>
      <div
        class="carousel-item"
      >
        <img
          alt="Fifth slide"
          class="d-block w-100"
          src="images/homepage/hpN11.jpeg"
        />
      </div>
      <div
        class="carousel-item"
      >
        <img
          alt="First slide"
          class="d-block w-100"
          src="images/new1.jpg"
        />
        <div
          class="carousel-caption"
        >
          <h3>
            First slide label, do you want this?
          </h3>
          <p>
            Do you want to add further comments below the label?
          </p>
        </div>
      </div>
      <div
        class="carousel-item"
      >
        <img
          alt="Second slide"
          class="d-block w-100"
          src="images/new2.jpg"
        />
        <div
          class="carousel-caption"
        >
          <h3>
            Second slide label, do you want this?
          </h3>
          <p>
            Do you want to add further comments below the label?
          </p>
        </div>
      </div>
      <div
        class="carousel-item"
      >
        <img
          alt="Third slide"
          class="d-block w-100"
          src="images/new3.jpg"
        />
        <div
          class="carousel-caption"
        >
          <h3>
            Third slide label, do you want this?
          </h3>
          <p>
            Do you want to add further comments below the label?
          </p>
        </div>
      </div>
      <div
        class="carousel-item"
      >
        <img
          alt="Forth slide"
          class="d-block w-100"
          src="images/new4.jpg"
        />
      </div>
      <div
        class="carousel-item"
      >
        <img
          alt="Fifth slide"
          class="d-block w-100"
          src="images/new5.jpg"
        />
      </div>
      <div
        class="carousel-item"
      >
        <img
          alt="Sixth slide"
          class="d-block w-100"
          src="images/new6.jpg"
        />
      </div>
      <div
        class="carousel-item"
      >
        <img
          alt="Sixth slide"
          class="d-block w-100"
          src="images/hoverdog.jpeg"
        />
      </div>
    </div>
    <a
      class="carousel-control-prev"
      href="#"
      role="button"
    >
      <span
        aria-hidden="true"
        class="carousel-control-prev-icon"
      />
      <span
        class="sr-only"
      >
        Previous
      </span>
    </a>
    <a
      class="carousel-control-next"
      href="#"
      role="button"
    >
      <span
        aria-hidden="true"
        class="carousel-control-next-icon"
      />
      <span
        class="sr-only"
      >
        Next
      </span>
    </a>
  </div>
  <div
    class="mainBody"
  >
    <div
      class="container"
    >
      <div
        class="row row"
      >
        <div
          class="leftCol col"
        >
          <h2
            class="bodyTitle"
          >
            Welcome to Salhouse Golden Retrievers
          </h2>
          <p>
            I am a dog lover who fell in love with this beautiful breed when I was a child in England. If you would like to read the story of the animals in my life please visit the 
            <a
              href="/mylifewithanimals"
            >
              My Life with Animals
            </a>
             page.
          </p>
          <p>
            I have had the privilege of sharing the lives of seven beautiful Golden Retrievers, most of whom are shown in the photos at the top of this page. Phoenix was my first Golden, and was the reason I wanted to have more of them. I can't thank her enough for the lessons she taught me and the loyalty only a dog can give even when the human is less than perfect.
          </p>
          <p>
            Kula was born in 2003 and was a really loving gentle dog. She was a great retriever, loving nothing more than to swim in the river or sea fetching as many sticks as we can throw for her. In 2007, Kula had a beautiful litter of puppies and we kept one female who we named Shadow.
          </p>
          <p>
            Shadow is a real character and is such fun to be with. She is so enthusiastic about her people and has the loveliest nature. Shadow had a litter of 7 puppies in April 2010 and we have kept one of those, Sherry. We added an item to the News page each day from the day the puppies were born to the time they left us. There are loads of photos and videos of these puppies and some stories of them growing up. Go to the News page to see these stories and more.
          </p>
          <p>
            Sherry is a very soft Golden, only wanting to please and be with us. She is a beautiful natured member of our family and dabbled in obedience for a while and did quite well. Sherry had a litter of two female puppies on 16 June 2015 - Autumn and Tuppence. Only having two in the litter made it impossible for us to part with one, so, although we had only wanted to keep one, we found ourselves with the two naughties!
          </p>
          <p>
            Tuppence is the smaller of the two sisters, but she is an exceedingly easy dog to train. She’s been taught lots of tricks and is a thinker. If you give her the right messages, she will eventually work out what you want her to do. She goes through her whole repertoire of tricks until she finds the one you want. I’ve never had a thinking dog like this before and she’s been really fun to train.
          </p>
          <p>
            Autumn is a relatively strong and powerful Golden girl, whose enthusiasm is very contagious and can be hard to contain when she’s out near water or when we’ve got a ball thrower. She is totally focused. Autumn had a litter of seven puppies in March 2019, of which we kept one who we called Siren.
          </p>
          <p>
            Siren is a darling. She’s a very showy girl and looks beautiful in the show ring and when out and about with her head up and tail flowing. She’s got a zany character and is another fun dog to be around.
          </p>
          <p>
            If you want to find out more about each of these beautiful dogs, please go to their pages to learn more. Shadow, Sherry and Autumn are breed champions. Sherry did compete in championship obedience competitions up to Test A level and has gained the CDX title. However, when Autumn and Tuppence came along it was too much to continue training Sherry and the puppies and so she is retired from the obedience ring. Tuppence is a very smart dog and at the present time is in training and hopefully one day I will get her into the obedience ring. Go to their individual pages to learn more about them.
          </p>
          <p>
            Goldens are wonderful pets - they are loyal, friendly and good with children . I hope you like mine and get your own one day!
          </p>
          <p>
            This website has lots of lovely photos of goldens and some other animals that I have owned, I hope you enjoy looking round it.
          </p>
        </div>
      </div>
      <hr
        class="sectionSeparator"
      />
    </div>
    <div>
      <div
        class="container"
      >
        <div
          class="row row"
        >
          <div
            class="leftCol col"
          >
            <p>
              Using images from this website:
            </p>
            <p>
              All photos on this website are copyright © by me. They are creative-commons licensed, you may use them under 
              <a
                href="https://creativecommons.org/licenses/by-nc-nd/3.0/nz/"
              >
                these non-commercial terms
              </a>
              .
            </p>
            <p>
              If you want to use them under different terms, please contact me at
              <a
                href="mailto:cmcnie@xtra.co.nz?subject=Enquiry regarding using photos from your Webpage"
              >
                 cmcnie@xtra.co.nz
              </a>
            </p>
            <a
              href="https://creativecommons.org/licenses/by-nc-nd/3.0/nz/"
            >
              <img
                class="bottom"
                src="images/nocc.png"
              />
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
